build_daemon/README.md

```diff
@@ -1 +1,6 @@
 This package provides support for running builds in a daemon.
+
+Note since `package:build_daemon` is a dependency of `package:build_runner`,
+if this is not a clean git checkout you should build this package with the following command:
+
+dart .dart_tool/build/entrypoint/build.dart.snapshot build --skip-build-script-check
```

Comment:
This file won't exist for everybody - they may need to at least do a `pub run build_runner generate-build-script` command and then I would recommend dropping the `.snapshot` here since that also won't be generated.

Reply:
I committed Nate's suggestion which addresses this.